java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestCommon.java

```diff
@@ -38,6 +100,10 @@ public static void initNNStreamer() {
             try {
                 Context context = InstrumentationRegistry.getTargetContext();
                 mInitialized = NNStreamer.initialize(context);
+
+                /* Copy nnstreamer folder in assets to the app's files directory */
+                mRootDirectory = getContext().getFilesDir().getAbsolutePath();
```

Comment:
Q: difference between getContext() and context = InstrumentationRegistry.getTargetContext(); (line 101)

Reply:
They're same. I changed the `getContext()` to `context`, Thanks!